pkg/lib/cockpit.js

```diff
@@ -1781,6 +1781,17 @@ function factory() {
                 options = { problem: options };
             if (!options)
                 options = { };
+
+            // Clean up all subscribers when closing
+            const subscriberKeys = Object.keys(subscribers);
+            for (const id of subscriberKeys) {
+                const subscription = subscribers[id];
+                if (subscription.callback) {
+                    send(JSON.stringify({ "remove-match": subscription.match }));
```

Comment:
I think the new Python implementation of cockpit-bridge doesn't actually implement "remove-match".

Reply:
> I think the new Python implementation of cockpit-bridge doesn't actually implement "remove-match".

It indeed does not, but all our docs still mention it, so we should properly deprecate this as the current state is confusing